content/app/guidelines/design/_index.nb.md

```diff
@@ -0,0 +1,69 @@
+5. **Tilgjengelighet for alle** Vi strekker oss langt for å lage digitale løsninger som skal være tilgjengelige for alle. Vi gjenbruker komponenter som en måte å optimalisere tilgjengelighet på - teknisk så vel som interaksjonsdesign og visuelt design.
+
+
+## Konsistens på tvers av tjenester
+Felles retningslinjer for design hjelper oss å skape konsistens på tvers av tjenester som lages i Altinn Studio. 
+
+Når brukere møter en ny tjeneste, har de med seg forventninger fra tidligere opplevelser. Når disse forventningene møtes, kreves det mindre av brukeren for å samhandle med komponentene, og det bygges selvtillit. Ved å ikke trenge å lære nye interaksjoner, vil brukerens fokus i større grad være på innholdet.
+
+Vi anbefaler **ikke** å endre CSS for individuelle tjenester som bruker Altinn-designet. Dersom vi åpner opp for designendringer og fargeendringer, vil dette skade konsistensen mellom tjenestene. Konsekvent design mellom tjenesteeiere er noe vi bør strekke oss mot. For å få det til kreves et godt samarbeid mellom fagmiljøet i Altinn og hos de ulike tjenesteeierene. **Vi ønsker oss en god felles løsning som kan brukes og forbedres av alle, fremfor mange forskjellige løsninger gjennom ulike CSS-endringer.** Dette vil også gjøre det lettere å kontrollere tilgjengeligheten på tvers av apper på plattformen vår. Velger du likevel å gå bort fra designet, er du selv ansvarlig for å følge alle [WCAG-krav](https://www.uutilsynet.no/wcag-standarden/wcag-20-standarden/86). 
+<br><br>
+
+{{% panel %}}**Har du behov for en ny komponent?**<p>Vi ønsker at alle er med på å videreutvikle <a href="/planlegging-og-design/guidelines/ui-komponenter/">biblioteket av komponenter</a> med både design og kode. I praksis vil dette si at Altinn ønsker å ta imot forslag til nye komponenter etter hvert som tjenesteeierne ser behovet for det. </p>
+
+<div class="a-btn-group" id="btnGroup">
+
+  <a href="https://github.com/Altinn/altinn-studio/issues/new/choose" class="a-btn mt-1 ">
+    Opprett en sak i github
+  </a>
+  <a href="https://altinn.slack.com/" class=" ">
+    Skriv til oss på Slack
+  </a>
+</div>
+
+{{% /panel %}}
+
+## Inngang til skjema
+Brukertester har vist at når brukerne starter på etaten/kommunens sider for å fylle ut et skjema og så blir sendt videre til Altinn, får de ofte opplevelsen av at «plutselig var jeg i altinn». Innganger til skjema bør derfor presenteres slik at det er tydelig at man går til Altinn, men samtidig slipper å gå innom unødvendige mellomledd. "Start innsending" bør lenke direkte til skjemaet. Opplevelsen til bruker vil da også i større grad bli at man får løst oppgaven i konteksten man er i, men at Altinn er brukt som løsning for innsending og historikk. 
+
+![Inngang til et skjema i Altinn bør presenteres tydelig](/app/guidelines/design/lenketilaltinn.png "Inngang til et skjema i Altinn bør presenteres tydelig")
```

Comment:
Kun filnavn på bilder, uten path. Er jo hele poenget med å ha foldere. Bruk engelsk i filnavn og `-`for mellomrom.

Reply:
Fikk det ikke til å fungere med kun filnavn.

Reply:
Interessant, ser ut til å skyldes at du ikke har lagt inn engelsk (som er default) oversettelse.
Om du legger til _index.en.md så fungerer det. Er det med oversettelser av dette nye innholdet noe som du har på blokka?

Reply:
Har fikset path og lagt til _index.en.md. på alle filer utenom ui-components, der fungerte det ikke.